intranet/static/js/ads.js

```diff
@@ -0,0 +1 @@
+var canRunAds = 1;
```

Comment:
This variable is checked in the login.html file to see of ads.js  and showads.js ran